package.json

```diff
@@ -4,29 +4,34 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "prebuild": "npm run clean && echo \"building...\"",
```

Comment:
Add the lint step?

`"prebuild": "npm run clean && npm run lint && echo \"building...\"",`

Reply:
yes that was my question, wasnt sure if we'd want that. for me i get a lot of lf <> crlf issues... maybe we should handle those at repo level not sure

Reply:
Oh sorry, I misunderstood and thought you just missed it.  I'm open either way, just think it is nice to have in general. So having it as part of the build step or incorporating it into a merge check in GH works for me.